File: app/src/main/java/mo/wall/org/dialog/MyDialogAcceptPar.java
package mo.wall.org.dialog;


import android.os.Parcel;
import android.os.Parcelable;


public class MyDialogAcceptPar implements Parcelable {

    public static final Creator<MyDialogAcceptPar> CREATOR = new Creator<MyDialogAcceptPar>() {
        @Override
        public MyDialogAcceptPar createFromParcel(Parcel in) {
            return new MyDialogAcceptPar(in);
        }

        @Override
        public MyDialogAcceptPar[] newArray(int size) {
            return new MyDialogAcceptPar[size];
        }
    };

    protected MyDialogAcceptPar(Parcel in) {
    }

    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
    }
}

File: app/src/main/java/mo/wall/org/dialog/MyDialogAcceptPar.kt
package mo.wall.org.dialog


import android.os.Parcel
import android.os.Parcelable


class MyDialogAcceptPar : Parcelable {

    protected constructor(`in`: Parcel) {

    }

    protected constructor() {

    }

    override fun describeContents(): Int {
        return 0
    }

    override fun writeToParcel(dest: Parcel, flags: Int) {}

    companion object {

        @JvmField
        val CREATOR: Parcelable.Creator<MyDialogAcceptPar> = object : Parcelable.Creator<MyDialogAcceptPar> {
            override fun createFromParcel(`in`: Parcel): MyDialogAcceptPar {
                return MyDialogAcceptPar(`in`)
            }

            override fun newArray(size: Int): Array<MyDialogAcceptPar?> {
                return arrayOfNulls(size)
            }
        }
    }
}
